UIManager.js

```diff
@@ -0,0 +1,248 @@
+import SquidexManager from "./SquidexManager.js";
+import CloudinaryManager from "./CloudinaryManager.js";
+
+export default class UIManager {
+    constructor(options) {
+        this.options = options;
+        this.formData = { codename: "", id: "", maxWidth: "", minWidth: "", width: "", height: "", crop: "", gravity: "", sourceTag: "" };
+        this.data = { assets: [], sources: [] };
+        this.sources = []; 
+        this.form = new SquidexFormField();
+        this.form.onInit(this.initializeSquidexManager.bind(this));
+    }
+
+    async initializeSquidexManager(ctx) {
```

Comment:
Refactor some of this code into the SquideManager along with this.form and updateValue. ( I can help you with this when you get to it. )

Reply:
done :)